templates/search/catalogue_search.html

```diff
@@ -25,11 +25,35 @@
     {% with search_tab=searchtabs.CATALOGUE.value %}
         {% include './blocks/search_results_hero.html' %}
 
+
+
         {% if buckets.current.result_count %}
-            <p class="search-results__explainer">Results for records held at The National Archives and other archives that match your search term.</p>
+
+                {% if form.group.value == 'tna' %}
+                    <p class="search-results__explainer">Results for records held at The National Archives that match your search term.</p>
+                {% endif %}
+
+                {% if form.group.value == 'digitised' %}
+                    <p class="search-results__explainer">Results for records available to download at The National Archives that match your search term.</p>
+                {% endif %}
+
+                {% if form.group.value == 'nonTna' %}
+                    <p class="search-results__explainer">Records held at other UK archives that match your search term.</p>
+                {% endif %}
+
+                {% if form.group.value == 'creator' %}
+                    <p class="search-results__explainer">A business, family, manor, organisation or person who has created records that match your search term.</p>
+                {% endif %}
+
+                {% if form.group.value == 'archive' %}
+                    <p class="search-results__explainer">Archives across the UK that match your search term.</p>
+                {% endif %}
```

Comment:
@JohnHeeryTNA , I think this is Ok for now since previous code is hard coded.

I think it would be better to have this refactored to pull out description from the Bucket 

see here example - tna, digitised:
- https://github.com/nationalarchives/ds-wagtail/blob/a909e04c2ce76a668304c9a9568c55329742c4c4/etna/ciim/constants.py#L122
- https://github.com/nationalarchives/ds-wagtail/blob/a909e04c2ce76a668304c9a9568c55329742c4c4/etna/ciim/constants.py#L122

I would suggest to create a follow on ticket BE update search results explainer.

Reply:
I'll raise a BE ticket for this Allan